test/test_braket_integration.jl

```diff
@@ -27,6 +27,18 @@ using Braket: I, name
     end
 end
 
+@testset "Circuit and Program Conversion" begin
+  qasm = """
+       qubit[2] q;
+       bit[2] c;
+
+       h q[0];
+       cz q[0], q[1];
+       measure q -> c;
+       """
+  convert(Braket.Circuit, BraketSimulator.Circuit(qasm))
```

Comment:
This could be a little more robust. Can we at least check the resulting instructions are correct?

Reply:
I am now checking if the resulting instruction are correct. 

Please note, that #46 fixes the failing test, as it allows instructions after measured gates